specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/virtualWan.json

```diff
@@ -0,0 +1,2105 @@
+          },
+          "204": {
+            "description": "No VpnSites exist by the name provided."
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/vpnSites": {
+      "get": {
+        "operationId": "VpnSites_ListByResourceGroup",
+        "x-ms-examples": {
+          "VpnSiteListByResourceGroup": { "$ref": "./examples/VpnSiteListByResourceGroup.json" }
+        },
+        "description": "Lists all the vpnSites in a resource group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource group name of the VpnSite."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. Returns the details of all the vpnSites in the resource group.",
+            "schema": {
+              "$ref": "#/definitions/ListVpnSitesResult"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/vpnSites": {
+      "get": {
+        "operationId": "VpnSites_ListBySubscriptionId",
+        "x-ms-examples": {
+          "VpnSiteList": { "$ref": "./examples/VpnSiteList.json" }
+        },
+        "description": "Lists all the VpnSites in a subscription.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. Returns the details of all the VpnSites in the subscription.",
+            "schema": {
+              "$ref": "#/definitions/ListVpnSitesResult"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualWans/{virtualWANName}/vpnConfiguration": {
```

Comment:
action name not indicative of an action. Should it be something like listvpnSiteConfiguration etc.?

Reply:
Yes, this has already been reviewe3d and signed-off by ARM few months, now just making this public. Here is the PR that was signed off by ARM: https://github.com/Azure/azure-rest-api-specs-pr/pull/372

Reply:
By action-name are you referring to the operationId?